CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 1.0.1 (under development)
 =========================
 
+Known Issues
+------------
+
+- In Firefox 55 (beta), participants who connect to an SFU Room will not see each
+  other's media Tracks. This does NOT happen in a P2P Room. Once we fix this issue,
```

Comment:
- Remove the word "media" in "media Tracks".
- Let's say "This issue does not impact Peer-to-Peer Rooms".
- For the last sentence, let's note "Our engineering team is already working on resolving this issue. Once corrected, customers should not need to upgrade the version of twilio-video.js used by their app to receive the fix (the issue is a back-end issue only)."